src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+.page {
+  display: flex;
+  height: 100vh;
+
+  &__calendar {
+    margin: auto;
```

Comment:
margin can't be all auto, add width and height margins

Reply:
ok